### ISSUE TITLE
App crashes on card type change

### ISSUE THREAD

**Issue by SlideAutoreport** [ccrama/Slide#247]
When I change the card layout in the settings to anything but the info bar or none,  the app crashes and the change is cancelled. 

Debug info:
 OS Version: 3.4.0-gf2fa664(448934.13)
 OS API Level: 21
 Device: htc_m8
 Model (and Product): HTC One_M8 (htc_europe)
 RELEASE: 5.0.1
 BRAND: htc
 DISPLAY: LRX22C release-keys
 CPU_ABI: armeabi-v7a
 CPU_ABI2: armeabi
 HARDWARE: qcom
 MANUFACTURER: HTC

Stacktrace: 

```java.lang.IllegalArgumentException: BİG_PİCTURE_CROPPED is not a constant in me.ccrama.redditslide.SettingValues$InfoBar
	at java.lang.Enum.valueOf(Enum.java:200)
	at me.ccrama.redditslide.SettingValues$InfoBar.valueOf(SettingValues.java:37)
	at me.ccrama.redditslide.Activities.EditCardsLayout$2$1.onMenuItemClick(EditCardsLayout.java:99)
	at android.support.v7.widget.PopupMenu.onMenuItemSelected(PopupMenu.java:250)
	at android.support.v7.internal.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:811)
	at android.support.v7.internal.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:153)
	at android.support.v7.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:958)
	at android.support.v7.internal.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:948)
	at android.support.v7.internal.view.menu.MenuPopupHelper.onItemClick(MenuPopupHelper.java:191)
	at android.widget.AdapterView.performItemClick(AdapterView.java:300)
	at android.widget.AbsListView.performItemClick(AbsListView.java:1247)
	at android.widget.AbsListView$PerformClick.run(AbsListView.java:3215)
	at android.widget.AbsListView.onTouchUp(AbsListView.java:4021)
	at android.widget.AbsListView.onTouchEvent(AbsListView.java:3793)
	at android.support.v7.internal.widget.ListViewCompat.onTouchEvent(ListViewCompat.java:120)
	at android.view.View.dispatchTouchEvent(View.java:8444)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2430)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2158)
	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2436)
	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2178)
	at android.widget.PopupWindow$PopupViewContainer.dispatchTouchEvent(PopupWindow.java:1682)
	at android.view.View.dispatchPointerEvent(View.java:8655)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4238)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4094)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3635)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3694)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3660)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:3773)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3668)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:3830)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3640)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:3694)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3660)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3668)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3640)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:5940)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:5908)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:5872)
	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6030)
	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:211)
	at android.os.MessageQueue.nativePollOnce(Native Method)
	at android.os.MessageQueue.next(MessageQueue.java:153)
	at android.os.Looper.loop(Looper.java:131)
	at android.app.ActivityThread.main(ActivityThread.java:5696)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1028)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:823)
```

**Comment by ccrama** [comment]
Believe it's fixed

**Comment by cem111** [comment]
Hey. It isn't fixed as of 4.2.5.5, though I found what causes it. My phone's language is Turkish which has both "İ" (lowercase i) and "I" (lowercase ı) in its alphabet.

See this in that debugging log right at the top: "BİG_PİCTURE_CROPPED." This should probably be "BIG_PICTURE_CROPPED."

When I change the OS language to English, the crashes stop.

Also related: Say you're subscribed to a sub that has an uppercase I in it, like IAmA. In the hamburger menu, sub's name gets lowercased to "ıama," which puts it at the bottom of the alphabetized list even though it should come before subs that start with "i."

So the problem seems to be that the app isn't playing nice with the Turkish language.
